### DISCUSSION TITLE
New font for tic-80

### DISCUSSION THREAD

**Discussion by icubyx** [nesbox/TIC-80#1711] [Ideas]
Hi, 
I'm new to the console and after fiddling around, I really am liking the console, but for the font. The wide font takes up too much space and the compact font is compact but could look better. Is it possible to change the font to the one used on the main website instead? It is compact as well as much more readable. 
Just my 2 bits and a request. 
Thanks for all the wonderful work being done on this - it has helped kindle my interest in learning programming!

**Comment by RobLoach** [comment]
You could use a sprite font...
https://github.com/nesbox/TIC-80/wiki/font

**Reply by icubyx** [reply]
That's great but I was suggesting the system font. Is there a way to change it?

**Reply by janAkali** [reply]
you can edit system fonts in the built-in sprite editor. First, use "config" command to load configuration, then make changes to sprites, edit colors, etc. and, finally, "save" your changes. 

You also can return everything to default with "config reset".

**Reply by joshgoebel** [reply]
Wait where in the sprite editor does one edit system fonts exactly?  This is news to me.

**Reply by Weeppiko** [reply]
You can load the config cart and edit whatever you want there, but this changes will only work for your own tic app, it can't be saved with carts

**Reply by joshgoebel** [reply]
Just to be clear the system font exist in RAM and can be modified while a cartridge is running and commands like print will use the system font from ram.

But this only affects the system font while the cartridge is running it doesn't affect Console etc.
